hugo/layouts/papers/single.html

```diff
@@ -143,15 +143,15 @@ <h5 class="card-title">Abstract</h5>
       <dt class="acl-button-row">Bib Export formats:</dt>
       <dd class="acl-button-row">
       {{ if (fileExists (printf "/data-export/%s" $bibfile)) }}
-        <a class="btn btn-secondary btn-sm" href="{{ $bibfile | relURL }}">BibTeX</a>
+        <a class="btn btn-secondary btn-sm" href="{{ $paper.url }}.bib">BibTeX</a>
       {{ end }}
       {{ $expfile := printf "/papers/%s/%s/%s.xml" (slicestr $volume_id 0 1) $volume_id $anthology_id }}
       {{ if (fileExists (printf "/data-export/%s" $expfile)) }}
-        <a class="btn btn-secondary btn-sm" href="{{ $expfile | relURL }}">MODS XML</a>
+        <a class="btn btn-secondary btn-sm" href="{{ $paper.url }}.xml">MODS XML</a>
       {{ end }}
       {{ $endfile := printf "/papers/%s/%s/%s.endf" (slicestr $volume_id 0 1) $volume_id $anthology_id }}
       {{ if (fileExists (printf "/data-export/%s" $endfile)) }}
-        <a class="btn btn-secondary btn-sm" href="{{ $endfile | relURL }}">EndNote</a>
+        <a class="btn btn-secondary btn-sm" href="{{ $paper.url}}.endf">EndNote</a>
       {{ end }}
```

Comment:
Use the canonical URL prefix instead of the deeply nested prefix.

Reply:
Same concern as with `$bibfile` above.

Reply:
So do you suggest that we generate `bib`, `endf`, etc links in the YAML, and only use those if present?

Reply:
This is what I ended up doing for simplicity.

Reply:
This is nice, but has the same issue with local browsing–the links will always point to the live server.